packages/crayons-core/src/components/popover/popover.tsx

```diff
@@ -142,9 +154,17 @@ export class Popover {
       this.fwShow.emit();
     }
   }
+  @Method()
+  async show() {
+    clearTimeout(this.timerId);
+    // timeout would be 0 until props is supplied explicit
+    this.timerId = setTimeout(() => {
+      this.triggerShowPopOver();
```

Comment:
Calling `triggerShowPopOver` with `0`  seconds inside `setTimeout` is not guaranteed to make function call immediately. Can we add a check if `showAfter` is not present and make a function call instead of invoking via setTimeout

Reply:
Done